code/ship/ship.cpp

```diff
@@ -11585,13 +11585,13 @@ int ship_fire_primary(object * obj, int force, bool rollback_shot)
 					points = num_slots;
 				}
 
-				if ( shipp->weapon_energy < points*winfo_p->energy_consumed*flFrametime || 
-					(winfo_p->wi_flags[Weapon::Info_Flags::Ballistic] && shipp->weapons.primary_bank_ammo[bank_to_fire] <= 0))
+				bool no_energy = shipp->weapon_energy < points* winfo_p->energy_consumed* flFrametime;
```

Comment:
The spacing was changed inconsistently here.

Reply:
I'm not sure I follow, what spacing are you referring to exactly?

Reply:
The spacing around the asterisk.  Spaces were added after them, but not before.  I'd be fine with either a space on each side or no space on either side, but not this one-sided spacing.

Reply:
Oh, I see. It must have been automatically when I pasted it back in...